src/client/version/client.go

```diff
@@ -67,6 +67,15 @@ func IsCustomRelease(version *pb.Version) bool {
 	return false
 }
 
+// Get branch from version string
+// patch release of .0 is always from the master. Others are from the M.m.x branch
+func BranchFromVersion(version *pb.Version) string {
+	if version.Micro == 0 {
+		return fmt.Sprintf("master")
+	}
```

Comment:
Since we branch after doing the .0 release, we will always have M.m.0 file in master. All subsequent 1.11.1, .2... will be in 1.11.x branch.

This is not ideal, but open to ideas.

Reply:
You're right, I forgot we had separate files per release on top of the branching, this is fine.